### DISCUSSION TITLE
v3.1.0

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#1748] [Versions]
Artisan v3.1.0 comes with renewed Bluetooth and MODBUS infrastructure and adds quite some additional device and machine support.

See the [detailed release history](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md) for a complete list of changes and fixes

---
PLEASE FOLLOW AND TAG US!  
&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.facebook.com/ArtisanScope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/facebook-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.instagram.com/artisanscope/"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/instagram-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/youtube-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://masto.ai/@artisan"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/mastodon-icon-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.linkedin.com/company/artisan-plus"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/linkedin-svgrepo-com.svg?raw=true" width="30"></a>

<hr /><em>This discussion was created from the release <a href='https://github.com/artisan-roaster-scope/artisan/releases/tag/v3.1.0'>v3.1.0</a>.</em>

**Comment by wvengen** [comment]
Great! :tada:
I've started work to release this on Flathub (to install the update on Linux distributions), and noticed that the metainfo file has version "3.1.ß", does this mean the Linux version is in beta, or is this a typo? (This will result in the Flatpak showing version "3.1.ß" as well.)

**Reply by MAKOMO** [reply]
Upps. This is a typo. Sorry. Could you please fix this one? THANKS!

**Reply by wvengen** [reply]
I'll correct it, and test the resulting Flatpak. Then afterwards, could we do a 3.1.1/3.1.2 release, so that there is a clear link between release version and tag in version control?

**Reply by MAKOMO** [reply]
I fear I do not have the energy to do a full 3.1.2 release right away. There are too many manual steps involved in creating and signing and notarizing all those builds. What about a just releasing that Flatpak as 3.1.1 with an increased build version 1 and a tag 3.1.1.1 on GitHub?

**Reply by wvengen** [reply]
I totally understand :) That sounds like a great idea, I'll do that when all is ready :+1:

**Reply by wvengen** [reply]
v3.1.1.1 was [released](https://github.com/flathub/org.artisan_scope.artisan/pull/8) [on Flathub](https://flathub.org/apps/org.artisan_scope.artisan). Wait a couple of hours, and you'll be able to install it.

**Reply by MAKOMO** [reply]
Excellent! Thanks for your support!

**Reply by AndroidCloned** [reply]
Report PDF is greyed out. On both mac and windows installs. 2.10.5 works as intended.

**Reply by roasterdave** [reply]
@AndroidCloned This is fixed on Windows in the [continuous build](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous).  It will be fixed on other platforms in a future build. 

Thanks for the report.  In the future, please start a new Discussion topic for new items and be specific about the version where you see a problem.  This topic is titled **"v3.1.0"**.  It took me time to confirm all the report menus work properly in v3.1.0 and then to find that the issue was new in the continuous builds v3.1.1.
